src/frontend/pages/blogs/edit-a-blog/EditBlog.js

```diff
@@ -226,12 +226,15 @@ const EditBlog = ({ add }) => {
           />
         </FormContainer>
 
-        <FormContainer title="Link (required)">
-          <TextInput
-            value={link}
-            onChange={setLink}
-            placeholder="https://..."
-          />
+        <FormContainer title="Content (required)">
+          <BlogInfo>
+            <TextInput
+              value={content}
+              onChange={setContent}
+              placeholder=""
+              multiLine={true}
+            />
+          </BlogInfo>
         </FormContainer>
```

Comment:
could you place the content above the date field?

Reply:
updated